core/src/main/scala/akka/persistence/cassandra/query/CassandraReadJournalConfig.scala

```diff
@@ -73,5 +76,15 @@ import scala.concurrent.duration._
     config.getDuration("events-by-tag.new-persistence-id-scan-timeout", MILLISECONDS).millis
   val eventsByTagOffsetScanning: FiniteDuration =
     config.getDuration("events-by-tag.offset-scanning-period", MILLISECONDS).millis
+  val eventsByTagCleanUpPersistenceIds: Duration = config.getString("events-by-tag.cleanup-old-persistence-ids") match {
+    case "off"       => Duration.Inf
```

Comment:
to be compatible with hocon boolean it should toLowerCase and `case "off" | "false"`

Reply:
:+1: